admin_guide/scoped_tokens.adoc

```diff
@@ -11,39 +11,43 @@
+scoped tokens.
 
+[[admin-guide-scoped-tokens-evaluation]]
 == Evaluation
-Scopes are evaluated by converting the set of scopes for a token into a set of PolicyRules.
-The request is then matched against those rules.  The request attributes must match at least 
-one of the scope rules to be passed to the "normal" authorizer for further authorization checks.
-
-
-== Scopes
-=== User Scopes
-User scopes are focused on getting information about a given user.  They are intent-based, so
-the rules are automatically created for you.
-
- 1. `user:info` - Allows access to information the user: name, groups, etc.
- 2. `user:check-access` - Allows access to self-localsubjectaccessreviews and self-subjectaccessreviews.
+Scopes are evaluated by converting the set of scopes for a token into a set of
+*PolicyRules*. Then, the request is matched against those rules. The request
+attributes must match at least  one of the scope rules to be passed to the
+"normal" authorizer for further authorization checks.
+
+[[admin-guide-scoped-tokens-user-scopes]]
+== User Scopes
+User scopes are focused on getting information about a given user. They are
+intent-based, so the rules are automatically created for you.
+
+* `user:info` - Allows access to information the user: name, groups, etc.
+* `user:check-access` - Allows access to *self-localsubjectaccessreviews* and *self-subjectaccessreviews*.
     Those are the variants where you pass an empty user and groups in your request object.
 
-=== Role Scope
-The role scope is about having the same level of access as a given role filtered by namespace.
+[[admin-guide-scoped-tokens-role-scope]]
+== Role Scope
+The role scope allows you to have the same level of access as a given role
+filtered by namespace.
 
- 1. `role:<cluster-role name>:<namespace or * for all>` - Limits the scope to the rules specified
+* `role:<cluster-role name>:<namespace or * for all>` - Limits the scope to the rules specified
     by the cluster-role, but only in the namespace specified.
-
-    Caveat: This prevents escalating access.  Even if the role allows access to resources like
-    secrets, rolebindings, and roles, this scope will *deny* access to those resources.  This helps
-    prevent unexpected escalations.  Many people do not think of a role like `edit` as being an escalating
-    role, but with access to a `secret`, it is.
-
- 1.  `role:<cluster-role name>:<namespace or * for all>:!` - Note the bang.  This is like #1, except that it
++
+Caveat: This prevents escalating access. Even if the role allows access to
+resources like secrets, rolebindings, and roles, this scope will *deny* access
+to those resources. This helps prevent unexpected escalations. Many people do
+not think of a role like *edit* as being an escalating role, but with access to
+a secret it is.
+
+* `role:<cluster-role name>:<namespace or * for all>:!` - Note the bang. This is similar to above, except that it
```

Comment:
Avoid starting sentences with "Note that..." or "Note the...". Suggest rephrase: This is similar to the example above, except that including the bang causes this scope to allow escalating access.